airbyte-integrations/connectors/destination-dynamodb/src/main/java/io/airbyte/integrations/destination/dynamodb/DynamodbDestination.java

```diff
@@ -27,7 +30,19 @@ public static void main(final String[] args) throws Exception {
   @Override
   public AirbyteConnectionStatus check(final JsonNode config) {
     try {
-      DynamodbChecker.attemptDynamodbWriteAndDelete(DynamodbDestinationConfig.getDynamodbDestinationConfig(config));
+      final DynamodbDestinationConfig dynamodbDestinationConfig =
+          DynamodbDestinationConfig.getDynamodbDestinationConfig(config);
+
+      // enforce ssl connection
+      if (!dynamodbDestinationConfig.getEndpoint().isBlank()) {
```

Comment:
Minor proposal to specify via method name what we are checking. 

  ```
  if (isNotSsl(dynamodbDestinationConfig.getEndpoint())) {
        return new AirbyteConnectionStatus()
            .withStatus(AirbyteConnectionStatus.Status.FAILED)
            .withMessage(NON_SECURE_URL_ERR_MSG);
    }
```
method:
```
  public boolean isNotSsl(String endpoint) {
    return !endpoint.isBlank() &&
        new URL(endpoint).getProtocol().equals(Protocol.HTTP.toString());
  }
```

Reply:
Good catch, updated. Thanks